clar2wasm/tests/standard/unit_tests.rs

```diff
@@ -811,7 +811,7 @@ fn test_lt_uint() {
     // 1 < 4294967297 is true
```

Comment:
Again, not related to this change, but I noticed while reviewing. This should be:
```suggestion
    // 1 < 0x1_0000_0000_0000_0001 is true
```
Can you update all of the comments that say `4294967296` or `4294967297`?

Reply:
yes, those aren't even the right numbers now that I'm looking at it...

Reply:
Yeah, forgot some bits 😀